polkadot/node/core/prospective-parachains/src/fragment_chain/mod.rs

```diff
@@ -689,19 +723,30 @@ impl FragmentChain {
 	}
 
 	/// Return a new [`CandidateStorage`] containing all the candidates from this `FragmentChain`,
-	/// as well as the unconnected ones.
-	pub fn as_candidate_storage(&self) -> CandidateStorage {
+	/// as well as the unconnected ones. This does not contain the candidates that used to be
+	/// pending availability.
+	pub fn advance_scope(&self) -> CandidateStorage {
```

Comment:
This function does not seem to actually advance the scope (move to the child relay chain block).